### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by ghost** [TheNewHEROBRINEX/Murder#56] [Ideas]
Change Emerald to Gold?
Added Interact Item Waiting Lobby, Player Spectator?
Added Custom Prize EndGame?

**Comment by TheNewHEROBRINEX** [comment]
The item to collect is currently emerald as it is on play.mcpz.net but I may consider adding an option in the config to customize it.
Please explain better what you mean with interact items in the waiting lobby.
I would like to implement spectator mode but currently, I can't find a good way to do that.
What types of prize would like to be implemented?

**Comment by ghost** [comment]
how about using libPiggyEconomy? it can support Economy Provider

**Comment by TheNewHEROBRINEX** [comment]
Ok thank you
